client/cli/src/bin/mint-rpc-client.rs

```diff
@@ -1,22 +1,48 @@
-use clap::Parser;
+use clap::{Parser, Subcommand};
 use fedimint_api::PeerId;
 use mint_client::api::WsFederationApi;
+use std::env;
 use url::Url;
 
 #[derive(Parser)]
-struct ApiCall {
-    url: Url,
-    method: String,
-    #[clap(default_value = "null")]
-    arg: String,
+#[clap(version)]
+struct Cli {
+    #[clap(subcommand)]
+    command: Commands,
+}
+
+#[derive(Subcommand)]
+enum Commands {
+    /// Print the latest git commit hash this bin. was build with
+    VersionHash,
+    ApiCall {
+        /// The url to use for the api call
+        url: Url,
+        /// The rpc method
+        method: String,
+        /// Args that will be serialized and send with the request
+        #[clap(default_value = "null")]
+        arg: String,
+    },
 }
 
 #[tokio::main]
 async fn main() {
-    let call = ApiCall::parse();
-    let arg: serde_json::Value = serde_json::from_str(&call.arg).unwrap();
-    let api = WsFederationApi::new(0, vec![(PeerId::from(0), call.url)]);
-    let response: serde_json::Value = api.request(&call.method, arg).await.unwrap();
-    let formatted = serde_json::to_string_pretty(&response).unwrap();
-    print!("{}", formatted);
+    let cli = Cli::parse();
+    match cli.command {
+        Commands::VersionHash => {
+            if let Ok(hash) = env::var("GIT_HASH") {
```

Comment:
In the app itself, we should only use the compiled time one only, no? `env!("GIT_HASH")`